python-sdk/src/astro/files/types/ndjson.py

```diff
@@ -76,14 +76,18 @@ def flatten(
             else:
                 extra_rows = []
 
-            df = pd.DataFrame(
-                pd.json_normalize([json.loads(row) for row in rows], **normalize_config)
+            # ToDo: Check if there is an alternative to pandas json_normalize(). A custom implementation is much faster.
+            # But that would entail breaking changes.
+            # ref : https://stackoverflow.com/questions/69945941/
+            #   why-does-pandas-json-normalize-run-slower-than-my-hand-crafted-loop
```

Comment:
Why not make this into an issue instead of having it as a TODO?

Reply:
@dimberman Added a ticket - https://github.com/astronomer/astro-sdk/issues/947